gui/src/components/Txs.tsx

```diff
@@ -28,7 +29,7 @@ export function Txs() {
 
   return (
     <div className="bg-white shadow sm:rounded-lg">
-      <ul role="list" className="divide-y divide-gray-200">
+      <ul className="divide-y divide-gray-200">
```

Comment:
@naps62 A `ul` already has the role of a list. Am I missing something?

Reply:
or maybe @zamith, our in-house accessibility advocate

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/list_role